src/components/DeprecatedBanner/DeprecatedBanner.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import styled from 'styled-components'
+import { ModalConsumer } from '../ModalManager/ModalManager'
+import { theme, Text, SafeLink, Button } from '@aragon/ui'
+import Banner from '../Banner/Banner'
+import Modal from '../Modal/Modal'
+
+const { negative, negativeText, accent, textDimmed } = theme
+
+const DeprecatedBanner = props => (
+  <ModalConsumer>
+    {({ showModal }) => <DeprecatedDao showModal={showModal} {...props} />}
+  </ModalConsumer>
+)
+
+class DeprecatedDao extends React.Component {
+  handleClick = () => {
+    const { dao, showModal } = this.props
+    showModal(DeprecatedModal, { dao })
+  }
+
+  render() {
+    const { children, dao } = this.props
+
+    return (
+      <React.Fragment>
+        <Banner
+          text="This Rinkeby DAO will be deprecated on 1/11/18"
+          textColor={negativeText}
+          onClick={this.handleClick}
+          buttonText="More info"
+          color={negative}
+          dao={dao}
```

Comment:
Do we need to pass `dao` as a prop here?

Reply:
That's the way I was able to get it all the way down [here](https://github.com/aragon/aragon/pull/371/files/4fd00eedf9d5809fa84e27f40b477e88a155f994#diff-e341e1f76a6ca8ec7fe005cd3ac11853R62)

Reply:
But I think `DeprecatedBody` gets rendered with the modal props, which is already handled [here](https://github.com/aragon/aragon/pull/371/files#diff-e341e1f76a6ca8ec7fe005cd3ac11853R24)?

The `Banner` doesn't seem to need the `dao` prop :).

Reply:
My bad. Thanks for following through.